### ISSUE TITLE
Modifications for user pages

### ISSUE THREAD

**Issue by alvarmaciel** [biomadeira/jasper#46]
Hi,
I want to use travis ci method for user pages. I'm testing and playing but i'm new with this. 
I made a branch "source" where i have the jasper files and now I'm building on "master" branch because i modify the destination_branch variable on Rakefile.
But is this the right procedure?

**Comment by alvarmaciel** [comment]
Ok, finally I can do it. 
1. Must have a "source" branch in wich i put de source
2. In RakeFile I chang the  destination_branch variable
3. In Travis CI set on the Build only if RakeFile is present (On the setting of your repo in travis)

**Comment by biomadeira** [comment]
@alvarmaciel yup! for personal pages jekyll uses *master* instead of *gh_pages* as the default branch. I guess it should work if you put your source in some other branch and then push the built pages to master...

**Comment by alvarmaciel** [comment]
Yes, I do that and change in the Rakefile de destination. Besides i have to change the config in Travis CI to disable the build in the branch that doesn't have a .travis.yml. That solve the attempt to build the master branch once the build source push to master.
